manifests/auth/flat_file.pp

```diff
@@ -1,12 +1,15 @@
-# Class: st2::auth::flat_file
+# == Class: st2::auth::flat_file
```

Comment:
Just curious if it's part of the documentation syntax/format and where I can read about it.